src/cli.ts

```diff
@@ -1,9 +1,9 @@
-import {search, SearchValue} from './search';
+import {SearchValue} from './search';
 
-const temp = 'require(\'http\')';
+const temp = 'const b = \'hihttp\';const a = require(\'http\');';
 run(temp);
 
 async function run(fileContent: string): Promise<SearchValue> {
-  const f = search(fileContent);
+  const f = require('./search').search(fileContent);
```

Comment:
It looks like we have some artifacts remaining from having rebased on top of `master`. Can you revert this file?

Reply:
done